guides/source/active_record_querying.md

```diff
@@ -1486,19 +1486,19 @@ books.each do |book|
 end
 ```
 
-The above code will execute just **2** queries, as opposed to **11** queries in the previous case:
+The above code will execute just **2** queries, as opposed to the **11** queries from the original case:
 
 ```sql
-SELECT `books`* FROM `books` LIMIT 10
+SELECT `books`.* FROM `books` LIMIT 10
```

Comment:
We should fix this same mistake on line 1415 in the `includes` section as well.

Reply:
Good catch.

Reply:
Fixed.